docs/api.md

```diff
@@ -5,7 +5,8 @@ The following extension commands are supported for programatic use. If a paramet
 
```

Comment:
Increment version?

Reply:
I prefer to do that closer to the release date (in the same PR where we update the changelog)